runtime/src/main/java/org/corfudb/runtime/collections/CorfuStreamEntries.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.runtime.collections;
+
+import lombok.Getter;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * A wrapper around a map of stream UUID -> List of CorfuStreamEntry objects
```

Comment:
Comment is unclear. What is the intention of this class and what is it going to be used for in expansion part.

Reply:
It is basically a wrapper over MultiSMRObject where updates from different tables are returned, grouped by their stream UUIDs. Let me try to reword it better.